.github/workflows/cdk-bootstrap.yml

```diff
@@ -144,18 +148,19 @@ jobs:
       - name: (Non-Prod) Configure AWS credentials
         uses: aws-actions/configure-aws-credentials@v1.6.1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ap-southeast-2
           mask-aws-account-id: true
-          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_NON_PROD }}
+          role-to-assume: arn:aws:iam::632223577832:role/AWS_ASSUME_ROLE_NON_PROD_OIDC
```

Comment:
Is there some way to de-duplicate this between jobs and workflows? I don't think GitHub workflows have a way to include a file with global variables, but I might be wrong.

Reply:
An idea that **might** work 

We could create a .env file and populate it with the necessary values (e.g. CiOidc=arn:aws..... NonProdOidc=arn:aws), and check this in to the .github/workflows directory

At the start of the workflow after repo checkout, we can run 
cat .env >> $GITHUB_ENV

Worth trying out?

Reply:
Sounds good!

Reply:
Updated in [00f8a46](https://github.com/linz/geostore/pull/1822/commits/00f8a46a054dd4c80906ac00fa73f511f3927a00).